ACD-4744: Created dockerfile for puppeteer

dwells-ancoris opened the pull request:
Creation of dockerfile for future use

dwells-ancoris committed:
ACD-4744: Created puppeteer docker image for repo

chris-ancoris commented on puppeteer/Dockerfile:
```diff
@@ -0,0 +1,18 @@
+
+ADD package.json package-lock.json /
+
+RUN npm install
```
I don't think we should add the package file here.  Instead I think we should simply have
```dockerfile
RUN npm install puppeteer
```
(as described on the [puppeteer web page](https://pptr.dev/).)

chris-ancoris commented on puppeteer/Dockerfile:
```diff
@@ -0,0 +1,18 @@
+FROM node:13.8-slim
```
Maybe better to use a LTS version?

dwells-ancoris committed:
ACD-4744: Updated puppeteer Dockerfile
Added puppeteer-with-jest Dockerfile
Added node_modules to .dockerignore
Removed package.json, package_lock.json
